### ISSUE TITLE
Return to work sound issue

### ISSUE THREAD

**Issue by kleinash** [schmich/marinara#77]
I love this app - it runs my day - looking forward to any updates. 

I would like to be able to set the sound for the end of my break to be louder. I hear the "take a break" sound just fine but for some reason, I am not hearing the "return to work" sound. I don't think it's working at the moment, but it could be that I am just not hearing it.

Mac Sierra 10.12.6
Chrome Version 61.0.3163.100 (Official Build) (64-bit)

**Comment by schmich** [comment]
@kleinash Thanks for the feedback.

In the settings, which sound are you using for the end-of-break audio notification?

I'm using various Creative Commons sounds that aren't normalized to the same volume. Some sounds are too quiet, some are too loud (see issue #71). I believe the fix is to just normalize the volume levels.

**Comment by kleinash** [comment]
@schmich At the moment its the battle horn, but I go to the ship bell sometimes. I haven't quite found a sound that I hear through my thoughts. Maybe I am looking for a sound that I have to shut off, like an alarm. So it will continue to sound until I get back to my work and shut it off.

**Comment by schmich** [comment]
I addressed volume issues in c1917a95c495e8a2fc20a68ec62ea57ec6e6ab2b. As I mentioned in #71, I don't think the results are perfect, but it should be better than before. I've also added a few more alarm sounds that are a bit more intrusive: analog alarm clock, digital watch alarm, and digital alarm clock. These changes will be present in the next release.

As for looping alarm sounds, I created #95 to track that feature.
